src/Traits/LanguageHydrationTrait.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace AllDigitalRewards\RewardStack\Traits;
+
+trait LanguageHydrationTrait
+{
+    private $lang;
+
+    /**
+     * @return string
+     */
+    public function getLang(): string
+    {
+        $lang = empty($this->lang) === true ? 'en' : $this->lang;
```

Comment:
this can be simplified to:

`return $this->lang`

default the property to 'en' and make sure in the setter that it can only receive 2 character codes. Will avoid the need to manage this complexity.